pages/common/zig.md

```diff
@@ -0,0 +1,36 @@
+# zig
+
+> The Zig compiler and toolchain.
+> More information: <https://ziglang.org/>.
+
+- Compile a project:
```

Comment:
Is a project in the working directory, or, does `zig` manages all projects and lets you choose which project to build? The same question is asked in the second to the fourth example.
```suggestion
- Compile a project in the current directory:
```